tools/supportedVersions.linq

```diff
@@ -291,8 +291,8 @@ public static class TextWriterExtensions
 
 public static class PackageMetadataResourceExtensions
 {
-	public static async Task<List<Version>> GetVersions(
-		this NuGetSearcher searcher, string packageId, ILogger logger, int majorOverlapYears, int minorOverlapMonths, List<Version> upstreamVersions, Dictionary<string, string> endOfLifePackages, int[] extendedSupportVersions)
+	public static async Task<List<UserQuery.Version>> GetVersions(
```

Comment:
interestingly, this seems to precicelsy undo Daniel's change made in https://github.com/Particular/docs.particular.net/commit/f9189a82d25f93a57525e11001123d045c118885#diff-df6876aa523e1bf3376a65ba59c1bfc7df3a18268c78da9b1ede4fbe13e7e93e 😕